soc/renesas/ra/ra8/bsp_mcu_family_cfg.h

```diff
@@ -0,0 +1,571 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BSP_MCU_FAMILY_CFG_H_
+#define BSP_MCU_FAMILY_CFG_H_
+#include "bsp_mcu_device_pn_cfg.h"
+#include "bsp_mcu_device_cfg.h"
+#include "../../../ra/fsp/src/bsp/mcu/ra8m1/bsp_override.h"
```

Comment:
Is it okay to configure the folder as `soc/renesas/ra/ra8`?
Looking at this line, I think `soc/renesas/ra/ra8m1` is better.

Reply:
@soburi : Thank you for your comment. We updated it. Currently, only ra8m1, we just keep "#include "bsp_override.h"
When others RA8 MCU are available, it will be:
```
#if defined(CONFIG_SOC_R7FA8M1AHECBD)
#include "ra8m1/bsp_override.h"
#include "ra8m1/bsp_mcu_info.h"
#define BSP_MCU_GROUP_RA8M1 (1)
#elif defined(CONFIG_SOC_R7FA8D1BHECBD)
#include "ra8d1/bsp_override.h"
#include "ra8d1/bsp_mcu_info.h"
```
Corresponding in hal_renesas, "fsp/ra/fsp/src/bsp/mcu" will be added into zephyr_include_directories